Dockerfile

```
@@ -0,0 +1,10 @@
+FROM golang:alpine as builder
+RUN mkdir /build
+ADD . /build/
+WORKDIR /build
+ARG opts
+RUN env ${opts} go build -ldflags="-w -s" -o cfn-format ./cmd/cfn-format/*
```

```suggestion
RUN env CGO_ENABLED=0 ${opts} go build -ldflags="-w -s" -o cfn-format ./cmd/cfn-format/*
```

Can we set CGO_ENABLED to 0 by default? Otherwise the default behaviour when building (without setting 'opts') will be to produce an unusable image.